test/unit/helper.spec.js

```diff
@@ -817,4 +817,12 @@ describe('Helpers Unit Tests', () => {
       );
     });
   });
+
+  describe('formatTimeTilMaturity()', () => {
+    it('should format blocks to human-readable time', () => {
+      const numBlocks = 463;
+      const time = helpers.formatTimeTilMaturity(numBlocks);
+      expect(time, 'to equal', '3 days and 5 hours');
+    });
+  });
```

Comment:
Nice test. Maybe add test cases for edge values like `undefined` and `0`.

Reply:
fixed